File: Example/NXNavigationExtensionDemo/Feature/Basic/Controllers/ViewController07_CustomBackButtonImage.m
//
//  ViewController07_CustomBackButtonImage.m
//  NXNavigationExtensionDemo
//
//  Created by Leo Lee on 2020/10/26.
//

#import <NXNavigationExtension/NXNavigationExtension.h>

#import "ViewController07_CustomBackButtonImage.h"

@interface ViewController07_CustomBackButtonImage ()

@end

@implementation ViewController07_CustomBackButtonImage

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (UIImage *)nx_backImage {
    return [UIImage imageNamed:@"NavigationBarBack"];
}

@end

File: Example/NXNavigationExtensionSwiftDemo/Basic/ViewController07_CustomBackButtonImage.swift
//
//  ViewController07_CustomBackButtonImage.swift
//  NXNavigationExtensionSwiftDemo
//
//  Created by Leo Lee on 2020/11/23.
//

import UIKit

class ViewController07_CustomBackButtonImage: BaseTableViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }
    
    override var nx_backImage: UIImage? {
        return UIImage(named: "NavigationBarBack")
    }

}
